### PR TITLE
Update push-content-type-to-hub.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a line about the changes that happen to the library view after content types are pushed to a document library.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7019

File: microsoft-365/contentunderstanding/push-content-type-to-hub.md [hunk 1]
```diff
@@ -38,6 +38,6 @@ To push content types to hubs
...
 
-After the settings are updated, the content type with these settings will be available in any newly associated site with the hub in a few minutes. Then, any new list or library created will have the content type automatically added to it within a few minutes of creation. A pushed content type will be added to a document library only if it derives directly or indirectly from the Document content type, and a content type will be added to a list only if it does not derive from the Document content type directly or indirectly.
+After the settings are updated, the content type with these settings will be available in any newly associated site with the hub in a few minutes. The default Library view will be changed to one of these automatically created views. Then, any new list or library created will have the content type automatically added to it within a few minutes of creation. A pushed content type will be added to a document library only if it derives directly or indirectly from the Document content type, and a content type will be added to a list only if it does not derive from the Document content type directly or indirectly.
```
Comment (JohanFreelancer9) [inline microsoft-365/contentunderstanding/push-content-type-to-hub.md]:
```suggestion
After the settings are updated, the content type with these settings will be available in any newly associated site with the hub in a few minutes. The default library view will be changed to one of these automatically created views. Then, any new list or library created will have the content type automatically added to it within a few minutes of creation. A pushed content type will be added to a document library only if it derives directly or indirectly from the Document content type, and a content type will be added to a list only if it does not derive from the Document content type directly or indirectly.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MikePlumleyMSFT Please merge this PR. Thanks

**Pull request merged**